mesa/examples/basic/conways_game_of_life/app.py

```diff
@@ -0,0 +1,71 @@
+import time
+
+import altair as alt
+import numpy as np
+import pandas as pd
+import streamlit as st
```

Comment:
Do we still want to use streamlit or update to Solara?

Reply:
in my view, we standardize all examples on solara